openpype/hosts/nuke/api/lib.py

```diff
@@ -2001,18 +2001,18 @@ def set_viewers_colorspace(self, viewer_dict):
                 "Attention! Viewer nodes {} were erased."
                 "It had wrong color profile".format(erased_viewers))
 
-    def set_root_colorspace(self, nuke_colorspace):
+    def set_root_colorspace(self, imageio_host):
         ''' Adds correct colorspace to root
 
         Arguments:
-            nuke_colorspace (dict): adjustmensts from presets
+            imageio_host (dict): adjustments from presets
 
         '''
-        workfile_settings = nuke_colorspace["workfile"]
+        workfile_settings = imageio_host["workfile"]
 
-        # resolve config data if they are enabled in host
+        # get config data if imageio is enabled
         config_data = None
-        if nuke_colorspace.get("ocio_config", {}).get("enabled"):
+        if imageio_host.get("enabled") and imageio_host["enabled"] is True:
```

Comment:
isn't this the same as just:
`if imageio_host.get("enabled") == True:`

Reply:
It should be if `imageio_host.get("enabled") is True`, but in this case only `if imageio_host.get("enabled"):` would work too.

Reply:
oh I see, `enabled` doesn't have a bool value